dataprofiler/labelers/labeler_utils.py

```diff
@@ -434,7 +434,7 @@ def get_config(self) -> Dict:
         base_config = super().get_config()
         return {**base_config, **config}
 
-    def reset_states(self) -> None:
+    def reset_state(self) -> None:
         """Reset states."""
```

Comment:
If we are going from plural to singular, I'd recommend updating the docstring as well.

Reply:
Cool, happy to fix the docstring

Reply:
thx